README.md

```diff
@@ -11,20 +11,22 @@ retrieved from an [`issue_comment` webhook event][issue_comment_wh]) for certain
 and rerunning jobs matching those commands.
 
 Some notes:
+
 - The PR either must have an `ok-to-test` label present on the PR, or the user who writes a command must
 be an organization member, or repo owner, contributor, or collaborator.
   - Typically `ok-to-test` can/should only be applied by repo reviewers/approvers to prevent spam and abuse.
 - `rerun-actions` should only be run on comment creation. See the below [examples](#examples) for how to do this.
 
 ## Comment commands
 
-These commands are named after
+The following commands are supported by this action:
+
+- `/rerun-all` - rerun all failed workflows.
+- `/rerun-workflow <workflow name>` - rerun a specific failed workflow.
 
```

Comment:
So the clarification below is part of the `/rerun-workflow` bullet
```suggestion
```

Reply:
SGTM - Also where did you see that you can't re-run a successful workflow. I briefly dived into that `s.RerunWorkflowByID(...)` and saw it's just making POST call to the API and I don't see anything in https://docs.github.com/en/rest/reference/actions#re-run-a-workflow that would call out that limitation.

Reply:
https://github.community/t/cannot-re-run-a-successful-workflow-run-using-the-rest-api/123661/4

It's worth linking ^ somewhere. Feel free to do that here, or I can in a follow up.

Reply:
Seems simple enough - I updated the README.md